pkgs/top-level/python-packages.nix

```diff
@@ -580,6 +580,8 @@ in {
 
   spglib = callPackage ../development/python-modules/spglib { };
 
+  sslib = disabledIf (!isPy3k) (callPackage ../development/python-modules/sslib { });
```

Comment:
disabledIf is intended for non-`buildPythonPackage` expressions. Pass `disabled = !isPy3k;` instead.

Reply:
Fixed, thanks.